### ISSUE TITLE
Featured images not working

### ISSUE THREAD

**Issue by stevepmitchell** [jonhoo/wp2ghost#11]
I saw a pull request that was added to support featured images, but it does not appear to work. I end up with imported content in Ghost that has no featured images. I'm not sure how I can verify if the data is in the JSON resultant file (I don't see anything). 

Yours is the closest script to export from Wordpress - it would be great if this could be fixed.

**Comment by jonhoo** [comment]
@stevepmitchell yeah, it unfortunately looks like @horacio abandoned #2. I'm no longer using WordPress *or* Ghost, but would be happy to take a look at a pull request adding this feature/an update to @horacio's.
